e2e/nomostest/new.go

```diff
@@ -467,6 +473,18 @@ func setupTestCase(nt *NT, opts *ntopts.New) {
 		}
 		nt.ociImages = nil
 	})
+	nt.T.Cleanup(func() {
+		for _, helmPackage := range nt.helmPackages {
+			remoteURL, err := nt.HelmProvider.PushURL(helmPackage.Name)
```

Comment:
Seems like the name of this method could be something better. It's being used for delete here, not just push.

Reply:
Perhaps ChartURL?

Reply:
The important distinction is that this is the URL that is fit for use from the test framework, as this returns the port forward from localhost. The other URL (SyncURL) returns the Service DNS from inside the cluster for use by helm-sync/oci-sync/git-sync. LMK if you have better ideas for a name

Reply:
I guess if we want to keep the method names similar for the helm provider and oci provider, then using "package" is ok. I think maybe we could refactor the names on the providers, but perhaps that should be a follow up PR to avoid making this one larger.

Reply:
maybe i'll take a a refactoring pass afterwards. it doesn't really need to block this fucntionality.